index.js

```diff
@@ -26,5 +26,14 @@ export default async function gitLog (options = {}) {
 
   args.push(options.range || 'HEAD')
 
+  if (options?.path) {
```

Comment:
`options` is always defined here

```suggestion
  if (options.path) {
```